webui.py

```
@@ -916,6 +925,47 @@ def refresh_files_clicked():
+                    # Update visibility when LoRA selection changes
+                    lora_model.change(
+                        fn=update_library_btn_visibility,
+                        inputs=[lora_model],
+                        outputs=[lora_library_btn],
+                        queue=False,
+                        show_progress=False
+                    )
+
+                    # Open library page when button is clicked
+                    lora_library_btn.click(
+                        fn=lambda x: None,
```

Ruff (0.14.5)

Unused lambda argument: `x`

(ARG005)